tracing-jersey/src/main/java/com/palantir/tracing/jersey/TraceEnrichingFilter.java

```diff
@@ -102,7 +101,15 @@ public void filter(ContainerRequestContext requestContext, ContainerResponseCont
         MultivaluedMap<String, Object> headers = responseContext.getHeaders();
         if (Tracer.hasTraceId()) {
             String traceId = Tracer.getTraceId();
-            Tracer.fastCompleteSpan(ContainerResponseContextTagTranslator.INSTANCE, responseContext);
+            Tracer.fastCompleteSpan(FunctionalTagTranslator.INSTANCE, sink -> {
+                sink.accept(TraceTags.HTTP_STATUS_CODE, Integer.toString(responseContext.getStatus()));
+                sink.accept(TraceTags.HTTP_URL_PATH_TEMPLATE, getPathTemplate());
+                sink.accept(TraceTags.HTTP_METHOD, requestContext.getMethod());
+                Object requestId = requestContext.getProperty(REQUEST_ID_PROPERTY_NAME);
```

Comment:
The undertow typed API attachments is rather nicer...

Reply:
agreed